settings.gradle

```diff
@@ -19,6 +19,9 @@ include ":JAICore:jaicore-search"
 
 
 // Software-Configuration
+include 'softwareconfigurationmodels'

```

Comment:
Still not convinced by this project name.

Reply:
didn't become better during the review?